common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -12,9 +12,11 @@ Fences
 [/site]
 
 [site wiki="plane"]
-ArduPilot supports failsafes generated by several types of GeoFences (boundaries described by latitude/longitude and/or altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. Types of GeoFences supported varies by vehicle.
+Plane firmware prior to 4.1  supports failsafes generated by a polygonal GeoFence (boundaries described by latitude/longitude and altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. See :ref:`plane:geofencing`
```

Comment:
```suggestion
Plane firmware prior to 4.1 supports failsafes generated by a polygonal GeoFence (boundaries described by latitude/longitude and altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. See :ref:`plane:geofencing`
```

Reply:
fixed